app/components/avatar-image-fallback/avatar-image-fallback.component.ts

```diff
@@ -22,31 +22,39 @@ class AvatarImageFallback implements AvatarImageFallbackBindings {
         this.imageClass = this.avatarClass
             ? this.avatarClass
             : 'app-avatar-image-fallback__default';
+
         this.setCharCount();
+        this.setLetterProfileAvatar();
+    }
 
-        this.$element.find('img').bind('error', () => {
+    //sets ngLetterAvatar character count; max 2 characters
+    setCharCount() {
+        if (this.name) {
+            this.charCount = this.name.includes(' ') ? 2 : 1;
+        }
+    }
+
+    setLetterProfileAvatar() {
+        // Use case: when user does not have profile image or name
+        if (!this.src && !this.name) {
             const appColors = ['#007BE3', '#FFC715', '#00D6B2', '#FF4848', '#385064'];
             const defaultFallback = angular
                 .element(document.createElement('i'))
-                .addClass(
-                    `fas fa-user app-avatar-image-fallback__default-user ${this.avatarClass}`
-                )
+                .addClass('fas fa-user app-avatar-image-fallback__default-user')
                 .css({
                     background: appColors[Math.floor(Math.random() * appColors.length)]
                 });
             this.$element.parent().prepend(defaultFallback);
             this.$element.remove();
-        });
-    }
-
-    setCharCount() {
-        if (this.name) {
-            this.charCount = this.name.includes(' ') ? 2 : 1;
+        } else {
+            this.$element.find('img').bind('error', () => {
+                this.$element[0].children[0].remove();
+            });
```

Comment:
so the broken image gets removed? However, does the letter avatar still show when an image fails to load?

Reply:
I added a new variable that will show the letter avatar when the image fails to load